### ISSUE TITLE
Using components that only have a template

### ISSUE THREAD

**Issue by Turbo87** [mitchlloyd/ember-islands#25]
I am using a few components that only have a template file, but no attached JS. Prior to Ember 2.7 and Ember Islands 1.0.1 this used to work fine, but now I'm getting errors like this:

```
Assertion Failed: ember-islands could not find a component named "flight-leg-panel" in your Ember appliction.

EmberError	@	ember-vendor-1797c179.js:20006
assert	@	ember-vendor-1797c179.js:7026
assert	@	ember-vendor-1797c179.js:19808
renderComponent	@	ember-vendor-1797c179.js:73659
didInsertElement	@	ember-vendor-1797c179.js:73626
trigger	@	ember-vendor-1797c179.js:45129
superWrapper	@	ember-vendor-1797c179.js:25682
Renderer.didInsertElement	@	ember-vendor-1797c179.js:15515
Renderer_dispatchLifecycleHooks	@	ember-vendor-1797c179.js:15417
Renderer_renderTopLevelView	@	ember-vendor-1797c179.js:15388
invoke	@	ember-vendor-1797c179.js:1299
flush	@	ember-vendor-1797c179.js:1365
flush	@	ember-vendor-1797c179.js:1173
end	@	ember-vendor-1797c179.js:487
run	@	ember-vendor-1797c179.js:609
join	@	ember-vendor-1797c179.js:629
run.join	@	ember-vendor-1797c179.js:24648
(anonymous function)	@	ember-vendor-1797c179.js:24711
```

**Comment by mitchlloyd** [comment]
Thanks for reporting! Do you mind checking this PR to make sure it solves your issue? You can use the branch name "half-components".

https://github.com/mitchlloyd/ember-islands/pull/26

**Comment by Turbo87** [comment]
@mitchlloyd that seems to work fine, thanks!

**Comment by mitchlloyd** [comment]
Fix released as 1.0.2.
